tests/test_backend.py

```diff
@@ -115,17 +115,31 @@ def test_rfc2047_helper(self):
     def test_send_mail(self):
         settings.AWS_SES_CONFIGURATION_SET = None
 
-        unicode_from_addr = 'Unicode Name óóóóóó <from@example.com>'
+        from_addr = 'Albertus Magnus <albertus.magnus@example.com>'
 
-        send_mail('subject', 'body', unicode_from_addr, ['to@example.com'])
+        send_mail('subject', 'body', from_addr, ['to@example.com'])
         message = self.outbox.pop()
         mail = email.message_from_string(smart_str(message['RawMessage']['Data']))
         self.assertTrue('X-SES-CONFIGURAITON-SET' not in mail.keys())
         self.assertEqual(mail['subject'], 'subject')
-        self.assertEqual(mail['from'], self._rfc2047_helper(unicode_from_addr))
+        self.assertEqual(mail['from'], self._rfc2047_helper(from_addr))
         self.assertEqual(mail['to'], 'to@example.com')
         self.assertEqual(mail.get_payload(), 'body')
 
+    def test_send_mail_unicode_body(self):
+        settings.AWS_SES_CONFIGURATION_SET = None
+
+        unicode_from_addr = 'Unicode Name óóóóóó <from@example.com>'
+
+        send_mail('Scandinavian', 'Sören & Björn', unicode_from_addr, ['to@example.com'])
+        message = self.outbox.pop()
+        mail = email.message_from_string(smart_str(message['RawMessage']['Data']))
+        self.assertTrue('X-SES-CONFIGURAITON-SET' not in mail.keys())
```

Comment:
It looks like this was spelled incorrectly?

Reply:
Good catch--you're not the first, it was fixed in this PR: #319

Reply:
ok good! sorry for the extra noise!